pvnet/models/multimodal/multimodal.py

```diff
@@ -124,15 +152,17 @@ def __init__(
             assert sat_history_minutes is not None
             assert nwp_forecast_minutes is not None
 
-            self.sat_sequence_len = (sat_history_minutes - min_sat_delay_minutes) // 5 + 1
+            self.sat_sequence_len = (
+                sat_history_minutes - min_sat_delay_minutes
+            ) // sat_interval_minutes + 1
 
             self.sat_encoder = sat_encoder(
                 sequence_length=self.sat_sequence_len,
                 in_channels=sat_encoder.keywords["in_channels"] + add_image_embedding_channel,
             )
             if add_image_embedding_channel:
                 self.sat_embed = ImageEmbedding(
-                    318, self.sat_sequence_len, self.sat_encoder.image_size_pixels
+                    image_embedding_dim, self.sat_sequence_len, self.sat_encoder.image_size_pixels
```

Comment:
Can we rename this to something like `num_embeddings` which would be in keeping with pytorch https://pytorch.org/docs/stable/generated/torch.nn.Embedding.html